script.subtitles.zimukux/CHANGES

```diff
@@ -1,3 +1,8 @@
+V0.1.5 (2022/4/2)
```

Comment:
this is not parsed by kodi, why not using a changelog.txt?

Reply:
removed as this is kept for the github project